File: mods/example/data/scripts/states/MainMenu.py
def create():
	print("system:Violet Engine has Python Support")

def postCreate():
	return
	sprite = NovaSprite(0, 300, Paths.image('icons/gf'))
	sprite.scrollFactor.set(1.5, 1.5)
	sprite.screenCenter(FlxAxes.X)
	add(sprite)

File: mods/example/data/scripts/states/MainMenu.lua
function create()
	print("system:Violet Engine has Lua Support")
end

--      "_" so it doesn't run
function _postCreate()
	local sprite = NovaSprite:new(0, 150, Paths.image('icons/darnell'))
	sprite.scrollFactor.set(1.5, 1.5)
	sprite.screenCenter(FlxAxes.X)
	add(sprite)
end
